### ISSUE TITLE
What needs to be done to add a new example?

### ISSUE THREAD

**Issue by Vaguery** [erp12/pyshgp#55]
I've got a couple of example problems I use in my GP classes and workshops, and they seem to be working already in Pysh. Aside from in-file docs, what needs to be added, and where, before I submit a PR?

**Comment by Vaguery** [comment]
See PR

**Comment by erp12** [comment]
I haven't thought about this yet, and I am open to suggestions.

I just pushed some changes to the examples and their documentation files that _maybe_ addresses this. It is the best idea I can think of off the top of my head.

The basic idea is there should be the problem `.py` file(s) and companion documentation `.rst` files. (which can be found in `docs/examples`). 

The `.rst` files should use references to the code file whenever possible to help keep both versions of the documentation in sync. This can be done with either `autodoc` (ie. `.. automodule:: examples.odd`) or something like this:

```
.. literalinclude:: /../examples/odd.py
        :pyobject: odd_error_func
```
If this approach is too complex, or somehow lacking, I am open to other ideas. Either way, this issue brings up a good point in that, there is no guide to contributing yet.

**Comment by erp12** [comment]
Addressed in #69
